great_expectations/data_context/data_context.py

```diff
@@ -155,13 +161,108 @@ def data_asset_name_delimiter(self, new_delimiter):
         else:
             self._data_asset_name_delimiter = new_delimiter
 
+    def get_validation_location(self, data_asset_name, run_id):
```

Comment:
this new method is logic for getting the location of validation result - refactored out from get_validation_result